### DISCUSSION TITLE
How to create a Refine application with Next.js appdir template and headless UI(no UI frameworks)?

### DISCUSSION THREAD

**Discussion by nishu-murmu** [refinedev/refine#4363] [Q&A] [answered]
When I looked at the documentations
there are two separate options for nextjs appdir and headless UI
For headless
`npm create refine-app@latest -- --example tutorial-headless`
For Next.js appdir
`npm create refine-app@latest -- --example with-nextjs-appdir`

Can we combine both?

**Comment by BatuhanW** [comment, marked as answer]
Hello @nishu-murmu, I'd suggest to start with `with-nextjs-appdir` example, and then you can check headless example and move it yourself. We don't have a combination of both but headless as the name implies, is headless and straightforward.
